cannot install npm-based plugin after installing a plugin from path

Considering installing 3 plugins:

1. kuzzle-plugin-auth-passport-local via npm
2. my-plugin via local path
3. kuzzle-plugin-logger via npm

We get the following output and Kuzzle does not start.

```
kuzzle    | ███ kuzzle-plugins: Downloading plugin:  kuzzle-plugin-auth-passport-local@2.0.1
kuzzle    | ███ kuzzle-plugins: Plugin my-plugin uses local plugin. Config will be overrided with local changes.
kuzzle    | ███ kuzzle-plugins: Plugin my-plugin is already installed. Skipping...
kuzzle    | ███ kuzzle-plugins: Plugin kuzzle-plugin-auth-passport-local@2.0.1 downloaded
kuzzle    | ███ kuzzle-plugins: Updating plugins configuration...
kuzzle    | ███ kuzzle-plugins: There is a problem with plugin kuzzle-plugin-logger. Check the plugin installation directory { [Error: Cannot find module '/var/app/node_modules/kuzzle-plugin-logger/package.json'] code: 'MODULE_NOT
_FOUND' }
kuzzle    | Cannot read property 'pluginInfo' of undefined
```

We can see from the logs that kuzzle-plugin-logger is not downloaded either.

--- a/lib/api/controllers/remoteActions/managePlugins.js
+++ b/lib/api/controllers/remoteActions/managePlugins.js
@@ -291,10 +291,12 @@ function createLock(pluginPath) {
  */
 function acquirePlugins(plugins) {
   var
-    installViaNpm = true,
     promises = [];
 
   _.forEach(plugins, (plugin, name) => {
+    var
+      installViaNpm = true;
+    
     if (plugin.path) {
       console.log('███ kuzzle-plugins: Plugin', name, 'uses local plugin. Config will be overrided with local changes.');
       installViaNpm = false;

